en/services/parachute.md

```diff
@@ -0,0 +1,158 @@
+
+
+#### PARACHUTE_ERROR_FLAGS
+[PARACHUTE_ERROR_FLAGS](#PARACHUTE_ERROR_FLAGS) is a bitmask that details the various critical errors that may occur within the parachute module or within the communication between the parachute and the drone. This is published in the [PARACHUTE_STATUS](#PARACHUTE_STATUS) message.
+
+A list of the various errors can be found below:
+| Value    | Parameter                                    | Description                                                                                                     |
+|----------|----------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
+| 1        | `PARACHUTE_ERROR_FLAGS_BAROMETER_ERROR`      | There is an error with the parachute barometer.                                                                  |
+| 2        | `PARACHUTE_ERROR_FLAGS_IMU_ERROR`            | There is an error with the parachute IMU.                                                                        |
+| 4        | `PARACHUTE_ERROR_FLAGS_RF_CONNECTION_ERROR`  | There is an error with the parachute's RF that is used for manual control.                                       |
+| 8        | `PARACHUTE_ERROR_FLAGS_LOW_POWER`            | Parachute module has low power.                                                                                  |
+| 16       | `PARACHUTE_ERROR_FLAGS_FC_CONNECTION_ERROR`  | There is an error with the connection between parachute and flight controller (FC).                              |
+| 32       | `PARACHUTE_ERROR_FLAGS_EFTS_CONNECTION_ERROR`| There is an error with the connection between parachute and Electrical Flight Termination System (EFTS).         |
+| 64       | `PARACHUTE_ERROR_FLAGS_POD_CONNECTION_ERROR` | There is an error with the parachute pod.                                                                        |
+| 128      | `PARACHUTE_ERROR_FLAGS_EFTS_DIAGNOSE`        | Parachute Electrical Flight Termination System (EFTS) diagnosis failed.                                          |
+| 256      | `PARACHUTE_ERROR_FLAGS_CHARGING_FAILED`      | Parachute module charging failed.                                                                                |
+| 512      | `PARACHUTE_ERROR_FLAGS_EXTERNAL_POWER_ERROR` | There is an error with the parachute external power source.                                                      |
+| 1024     | `PARACHUTE_ERROR_FLAGS_GS_CONNECTION_ERROR`  | There is an error with the connection between parachute and Ground Station (GS).                                 |
+| 2048     | `PARACHUTE_ERROR_FLAGS_GPS_ERROR`            | There is an error with the parachute's GPS.                                                                      |
```

Comment:
does the parachute have a GPS? Do they have other variants that have a GPS?

Reply:
Guessing they do, and they use it for geofencing? Is that what we plan to do for EU?